pom.xml

```
@@ -13,12 +13,12 @@
     <version>${revision}-${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
-        <revision>6.8.1</revision>
+        <revision>6.9.2</revision>
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <jenkins.version>2.401.1</jenkins.version>
```

The build fails because this needs to be updated to 2.401.3.  The dependency on 2.xxx.1 is usually temporary until the release of the 2.xxx.3.  In this case, the dependency has lasted longer.
```suggestion
        <jenkins.version>2.401.3</jenkins.version>
```